version-control/git/github/instructor_notes.md

```diff
@@ -1,5 +1,13 @@
 # git/GitHub
 
+<!-- 
+
+This file looks like needs to be updated to correspond to the current order of the 
+curriculum.   We now introduce github after "local", "Make Incremental Changes 
+I and II."  This introduces github at the beginning.
+
+-->
```

Comment:
I don't think I've ever looked at/touched the instructor notes (for better or for worse...); feel free to update them if you'd like to!